target/linux/ath79/nand/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,11 @@ ath79_setup_interfaces()
 	aerohive,hiveap-121)
 		ucidef_set_interface_lan "eth0"
 		;;
+	domywifi,dw33d)
+		ucidef_set_interfaces_lan_wan "eth1.1" "eth0.2"
```

Comment:
This is unnecessary. ucidef_add_switch should handle lan and wan setup properly.